test/bson_older_versions_tests.js

```diff
@@ -73,18 +76,160 @@ describe('Current version', function () {
         done();
       });
     });
+  });
 
-    it(`serializes correctly against ${version} Binary class`, function () {
-      const oldBSON = require(getImportPath(version));
-      const binFromNew = {
-        binary: new newBSON.Binary('aaaa')
-      };
-      const binFromOld = {
-        binary: new oldBSON.Binary('aaaa')
-      };
-      expect(oldBSON.prototype.serialize(binFromNew).toString('hex')).to.equal(
-        newBSON.serialize(binFromOld).toString('hex')
+  // Node.js requires an .mjs filename extension for loading ES modules.
+  before(() => {
+    try {
+      fs.writeFileSync(
+        './bson.browser.esm.mjs',
+        fs.readFileSync(__dirname + '/../dist/bson.browser.esm.js')
       );
-    });
+      fs.writeFileSync('./bson.esm.mjs', fs.readFileSync(__dirname + '/../dist/bson.esm.js'));
+    } catch (e) {
+      // bundling fails in CI on Windows, no idea why, hence also the
+      // process.platform !== 'win32' check below
```

Comment:
Lmk if there is a ticket for this so I can mention it here, I couldn’t find any, but this issue doesn’t appear to be new.

Reply:
I do not see one already but I think this fine to leave out, since we test the functionality of the bson lib on windows this bundle compatibility seem adequately covered if it runs on at least linux. We can always file a ticket to track but I don't think we'll need to look into this